provision-osx/tasks/configure_buildfarm_proxy.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+-
+  name: Set node proxy configuration with authentication
+  shell: "networksetup -{{item}} {{proxy_device}} {{proxy_url}} {{proxy_port}} on {{proxy_user}} {{proxy_pass}}"
+  with_items: "{{proxy_ctx}}"
+  become: true
+  when: proxy_user is defined and proxy_pass is defined
+
+-
+  name: Set node proxy configuration without authentication
+  shell: "networksetup -{{item}} {{proxy_device}} {{proxy_url}} {{proxy_port}}"
```

Comment:
we don't have proxy_url specified anywhere in digger_installer. It is created in the proxy_setup in rhmap_ansible but will need to be specified somewhere for digger installer to work independently. I think it's sufficient to add it to the sample inventory file and give an example of the syntax. Or alternatively we could move the proxy_setup role to the digger-installer. The reason for not including it here was that we may have wanted to use it with other playbooks elsewhere but I'm not sure if we will do that now.